### PR TITLE
Add eslint and prepublish process, convert all dev codes to ES6, remove default /api/upload-trace

### PR DESCRIPTION
**Pull request by kingiw**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Ref: https://www.jianshu.com/p/91e764a1bedc
+module.exports = {
+    env: { //环境
```
Comment (hsluoyz) [inline .eslintrc.js]: English only.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Ref: https://www.jianshu.com/p/91e764a1bedc
```
Comment (hsluoyz) [inline .eslintrc.js]: Remove Chinese blog reference, if necessary, change to a English blog.

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    parserOptions: {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    rules: { 
+        'indent': [0, 4],
+        'semi': [1, 'always'],
+    }
+};
```
Comment (hsluoyz) [inline .eslintrc.js]: new line

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mouselog",
-  "version": "0.1.2",
+  "version": "0.1.1-beta1",
```
Comment (hsluoyz) [inline package.json]: Why version becomes lower? You should squash or rework your commits.

File: src/config.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-const urljoin = require('url-join');
-const debug = require('./debugger');
+import urljoin from 'url-join';
+import * as debug from './debugger';
 class Config {
```
Comment (hsluoyz) [inline src/config.js]: new line

File: .eslintrc.js (hunk 2 above)
Comment (kingiw) [reply .eslintrc.js]: This is outdated. I have removed it on a newer commit

File: .eslintrc.js (hunk 1 above)
Comment (kingiw) [reply .eslintrc.js]: This is outdated. I have removed it on a newer commit

**Comment by kingiw** [comment]
PR is updated, plz review

**Comment by hsluoyz** [comment]
lgtm

**Pull request merged**